ovsdb/client.go

```diff
@@ -139,14 +161,28 @@ func (c *Client) rpc(ctx context.Context, method string, out interface{}, args .
 		Response: ch,
 	})
 
+	// Ensure that the callback is always cleaned up on return from this function.
+	// Note that this will result in the callback being deleted twice if the RPC
+	// returns successfully, but that's okay; it's a no-op.
+	//
+	// TODO(mdlayher): a more robust solution around callback map modifications.
+	defer func() {
```

Comment:
Like it!  I believe you mentioned about the clean up code in the producer side, and I don't see the remove diff in this PR.  Does that already handled by other PR?  Just want to double check.

But with the iteration spirit, LGTM. :)